Information/CenterOfMass.FCMacro

```diff
@@ -571,8 +571,8 @@ class CenterofmassWidget(QtGui.QWidget):
             density = self.materials[sol].spin.value()
             densityU = self.comboUnit.currentText()
             densityConverted = float(self.tu('%f %s' % (density, densityU)) / self.tu('kg/m^3'))
-            dThous = densityConverted // 1000
-            dHund = (densityConverted - dThous*1000) // 100
+            dThous = densityConverted // 1000.
```

Comment:
'//' is the entire division and should be replaced with '/'.

Reply:
@chupins was this addressed?

Reply:
The entire division is done on purpose. 
It has been done to have a color difference between close density materials.

Reply:
OK, sorry about the noise.